src/resolvers/resolverUtils.ts

```diff
@@ -0,0 +1,76 @@
+import { lookupUser, lookupId, lookupCourses, lookupSchedule } from '../prisma';
+import { User, CourseSection, Schedule, CourseId, Term, Role } from '../schema';
+export { getMe, getUserByID, getCourses, getSchedule };
+
+const failedMeandID = {
+  id: 0,
+  username: 'Not Found',
+  password: 'Not Found',
+  role: Role.User,
+  preferences: [],
+  active: false,
+};
+
+const failedSchedule = {
+  id: 'Not Found',
+  year: 0,
+  createdAt: Date,
+  courses: [],
+};
+
+async function getMe(username: string): Promise<User> {
```

Comment:
You should pass in the user object from the session and parse out from that instead of doing another database lookup.

Pass in the ctx to this instead of the username and get the user with `ctx.session.user`